test/hc_e2e.sh

```diff
@@ -2,14 +2,17 @@
 #This helper script will run E2E UI tests for Hub Central (https://github.com/marklogic/marklogic-data-hub/tree/develop/marklogic-data-hub-central/ui/e2e)
 
 #override java and node version for Jenkins
-[[ $USER = 'builder' ]] && { export PATH=/home/builder/java/jdk1.8.0_201/bin:/home/builder/nodeJs/node-v16.19.1-linux-x64/bin/:$PATH; unset JAVA_HOME; }
+[[ $USER = 'builder' ]] && { export PATH=/home/builder/java/jdk1.8.0_201/bin:/home/builder/nodeJs/node-v14.15.4-linux-x64/bin/:$PATH; unset JAVA_HOME; }
```

Comment:
node.js 14 is pretty old couldn't we use a more recent release ?

Reply:
The tests complained about compatibility so I lowered it. I believe the cypress version is pretty old.